### DISCUSSION TITLE
We have both `main` and `master`. Let's drop both and just rely on a global `next` + version-specific `next`s.

### DISCUSSION THREAD

**Discussion by dead-claudia** [MithrilJS/mithril.js#2952] [Meta/Feedback]
`main` was (until about a couple hours ago) only one commit behind `master`, and I recently fixed that. Let's complete the switch from `master` to `main` and delete `master`, so we have only one branch.

Though...maybe we should just drop `master`/`main` entirely and rely on the version branches instead. It'd be a lot simpler to work with, and it'd avoid some future confusion. I'm imagining the following system:

- Branch `release-v{0,1,2,...}` points to the commit used to generate the latest release for that major version. These would only be updated on publish.
- Branch `next-v{0,1,2,...}` points to the tip of tree for docs for that version, including unreleased updates.
- Branch `next` points to the tip of tree. A GH Action will update `next-vN` where `N` is the current major version in `next`, so it can be a one-time setup. If it can't fast-forward (preferred) or rebase against the current `next-vN`, I'll have it create a pull request.
- Branch `docs` points to the tip of tree for the docs. Updates to `docs/` in this branch sync to `next` (in the same way `next` syncs to `next-vN`, but updates to `docs/` in `next` do *not* sync to this.
- Branch `gh-pages` will see updates automatically from `docs` and (to the archive only) `release-vN`.

**Comment by dead-claudia** [comment]
cc @MithrilJS/collaborators

**Comment by JAForbes** [comment]
Yeah agreed

**Comment by dead-claudia** [comment]
And completed.
